cmd-args.md

```diff
@@ -30,15 +30,15 @@ When you create a {{site.data.keyword.codeengineshort}} application or job, you
 | The arguments that are passed to the command.    | `CMD`    | `args` |
 {: caption="Docker and {{site.data.keyword.codeengineshort}} names" caption-side="bottom"}
 
-- If `--command` is used, then any image `Entrypoint` value is overwritten and any image `cmd` values are ignored.
-- If `--argument` is used, then any image `Command` value in overwritten.
+- If `--command` is used, then any image `Entrypoint` value is overwritten and any image `CMD` values are ignored.
+- If `--argument` is used, then any image `CMD` value is overwritten.
```

Comment:
Same like above. Let's stick to the term that this page uses. Thanks for spotting the typo in this line!
```suggestion
- If `--argument` is used, then any image `Command` value is overwritten.
```